core/environment.go

```diff
@@ -244,3 +265,7 @@ func (env *Env) ResolveSymbol(s Symbol) Symbol {
 		ns:   vr.ns.Name.name,
 	}
 }
+
+func init() {
+	GLOBAL_ENV.SetCurrentNamespace(GLOBAL_ENV.EnsureNamespace(MakeSymbol("user")))
```

Comment:
I am not seeing a corresponding removal of a similar line anywhere in the diff, so not sure what it's for...

Reply:
> I am not seeing a corresponding removal of a similar line anywhere in the diff, so not sure what it's for...

`NewEnv()` used to do this (to `currentNs`), though it's not immediately obvious -- see the combination of the `NewEnv()` call in the `var GLOBAL_ENV` definition and line 88 (I think it is) in the old `NewEnv()`.

Without this line (all of `func init()` here, that is), the build quickly crashes.

Reply:
Gotcha!